### ISSUE TITLE
Please publish a version for Scala 2.13.0-RC1

### ISSUE THREAD

**Issue by Sciss** [dispatch/reboot#210]
it's the only dependency in my huge tree still missing :) thx

**Comment by SethTisue** [comment]
just to see what the next step here might be, I tried `++2.13.0-M5!` followed by `Test/update` and got

```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: ws.unfiltered#unfiltered-netty-server_2.13.0-M5;0.9.1: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
```

**Comment by farmdawgnation** [comment]
Yeah, until unfiltered is up and running or someone rewrites our tests, we're stuck.

**Comment by SethTisue** [comment]
I see. noted at scala/make-release-notes@0f6d154a99cc18e7c3e9d92bf5227421297787e5

**Comment by farmdawgnation** [comment]
The Lift Committers are open to a PR, but converting Lift to the new collection lib could be an undertaking. We’ve all been busy with life outside open source.

**Comment by Sciss** [comment]
ping - Scala 2.13.0-RC1 is out

**Comment by Sciss** [comment]
Note, I can update https://github.com/dispatch/reboot/pull/211/ for Scala 2.13.0-RC1 if you agree - it just drops Lift support under 2.13.x

**Comment by mfirry** [comment]
ping - Scala 2.13.0 is out

**Comment by mfirry** [comment]
I guess unfiltered is (almost) done: https://github.com/unfiltered/unfiltered/commit/8e5b183c8966485b1a755c7f02ec8b3aea399380

**Comment by SethTisue** [comment]
I opened an unfiltered ticket, https://github.com/unfiltered/unfiltered/issues/455

**Comment by SethTisue** [comment]
Unfiltered has published. see linked ticket

**Comment by xuwei-k** [comment]
https://github.com/dispatch/reboot/pull/203

**Comment by farmdawgnation** [comment]
With @xuwei-k's PR in place I'll work on spinning a build for this in the next few days. We'll roll without lift-json for now.

**Comment by farmdawgnation** [comment]
Dispatch 1.1.0 was released this evening with support for Scala 2.13

**Comment by SethTisue** [comment]
awesome, thank you Matt!
